railgun/manager/manager.go

```diff
@@ -19,15 +32,17 @@ import (
 	kongctrl "github.com/kong/kubernetes-ingress-controller/railgun/controllers/configuration"
 	"github.com/kong/kubernetes-ingress-controller/railgun/controllers/corev1"
 	"github.com/kong/kubernetes-ingress-controller/railgun/internal/ctrlutils"
-	"github.com/spf13/pflag"
-	"k8s.io/apimachinery/pkg/runtime"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
-	"k8s.io/client-go/tools/clientcmd"
-	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/healthz"
-	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var (
+	// Release returns the release version
+	Release = "UNKNOWN"
+
+	// Repo returns the git repository URL
+	Repo = "UNKNOWN"
+
+	// Commit returns the short sha from git
+	Commit = "UNKNOWN"
```

Comment:
// The value of X is set by the build process at compile time.

Reply:
:+1: 8f21594a7804f638b7e690259ba128efc318cb8d